README.md

```diff
@@ -38,7 +38,7 @@ object specifying the following three options:
  * retries: Maximum number of retries in acquiring a lock if the first attempt failed (default: 0)
  * delay:   Time in milliseconds to wait between each attempt (default: 50 ms)
 
-``` javascript
+```javascript
 const Promise = require('bluebird');
```

Comment:
still references bluebird which is no longer part of the package

Reply:
This PR was opened before all the recent updates, so I guess it's not needed anymore and can be closed :)